src/Illuminate/Database/Migrations/MigrationRepositoryInterface.php

```diff
@@ -57,6 +62,20 @@ public function createRepository();
     public function repositoryExists();
 
     /**
+     * Determine if the migration repository has the tag column.
+     *
+     * @return bool
+     */
+    public function repositoryTagColumnExists();
```

Comment:
You could just prefix the migration name with tag/ in the existing column. Would be a slightly more complicated query but would be BC.

```
tag/2016_01_01_migration_name
```